### DISCUSSION TITLE
Change Url

### DISCUSSION THREAD

**Discussion by umeshpunia** [AzuraCast/AzuraCast#5682] [Q&A]
Hi All
I successfully installed azuracast all thanks to documentation
My Question Is Like my domain is xyz.com
when i open my domain azuracast open
but i want to open my custom site and when i open xyz.com/server then azuracast open
how to achieve like this please help
thanks

**Comment by BusterNeece** [comment]
@umeshpunia AzuraCast isn't really designed to run in a subfolder, but it can easily run on a subdomain if you just update the DNS records of your domain to point an A record for a subdomain to the IP address of your AzuraCast server.

**Reply by umeshpunia** [reply]
then how to achive this?
